benchmark/process/next-tick-depth.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
-var common = require('../common.js');
-var bench = common.createBenchmark(main, {
+
```

Comment:
It stumbled over it while I cherry-picked this part from the other PR due to a conflict in this file. I can keep it as it is if you want me to undo it.